spec/cypress/app_commands/scenarios/admin/school_participants.rb

```diff
@@ -47,18 +47,13 @@
 
 npq_user = FactoryBot.create(:user, full_name: "Natalie Portman Quebec", email: "natalie.portman@quebec.ca")
 
-npq_profile = FactoryBot.create :participant_profile,
-                                :npq,
-                                validation_data: nil,
-                                user: npq_user,
-                                created_at: Date.parse("19/09/2019")
-
-FactoryBot.create :npq_validation_data,
-                  id: npq_profile.id,
-                  user: npq_user,
-                  date_of_birth: Date.parse("10/12/1982"),
-                  nino: "NI123456",
-                  created_at: Date.parse("19/09/2019"),
-                  teacher_reference_number: "9780824"
-
-npq_profile.update!(updated_at: Date.parse("19/09/2019"))
+npq_validation_data = FactoryBot.create :npq_validation_data,
+                                        id: npq_profile.id,
```

Comment:
My bad, after removing npq profile above we should probably remove this line and let the db auto-generate it. 

```
Body: Puma caught this error: undefined local variable or method `npq_profile' for CypressOnRails::CommandExecutor:Class (NameError)
```
is failing :(

Reply:
ok should be sorted on next run. nah, it's my fault for not checking